micro-rdk/src/common/grpc.rs

```diff
@@ -1239,7 +1279,7 @@ where
     }
 
     #[cfg(feature = "camera")]
-    fn camera_get_image(&mut self, message: &[u8]) -> Result<(), ServerError> {
+    fn camera_get_image(&mut self, message: &[u8]) -> Result<Bytes, ServerError> {
```

Comment:
Does this compile? The implementations still return an `Ok(())`.
The camera rpc handlers don't call `encode` directly to reduce copying though that might be unavoidable.

Reply:
now yes :P i am returning the encoded Bytes does it looks good to you?